### PR TITLE
chore: updates optic dependencies

### PR DESCRIPTION
**Pull request by gablaxian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,12 @@
   ],
   "resolutions": {
     "urijs": "^1.19.11",
-    "vm2": "3.9.18"
+    "@useoptic/optic/conf/semver": "7.5.2",
+    "@useoptic/optic/postman-collection/semver": "7.5.2",
+    "@useoptic/optic/latest-version/**/semver": "6.3.1",
+    "@useoptic/optic/update-notifier/semver-diff/semver": "6.3.1",
+    "@useoptic/optic/update-notifier/configstore/**/semver": "6.3.1",
+    "@useoptic/optic/is-elevated/**/semver": "5.7.2"
```
Comment (gergo-papp) [inline package.json:82]: @gablaxian  why was this necessary?

Comment (gablaxian) [reply package.json:82]: @gergo-papp To address High severity issues in those indirect deps.
